### PR TITLE
Add Device Farm docs for alexa workshop.

### PR DESCRIPTION
**Pull request by fanyizhe**
*Issue #, if available:*

*Description of changes:*
Add Device Farm docs for alexa workshop.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: content/getting-started/device-farm-tutorial.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+{{% notice info %}}
+In this section we will show you how to add Google account and install Alexa App on a remote Android device using AWS Device Farm. You can also start an IOS device, but you can not use Apple Store to install Apps or add your Apple ID to this device due to security reasons. You can only use the upload option to upload and install your apps. So we can not install Alexa app on IOS devices.
+{{% /notice %}}
+
+{{% notice info %}}
+AWS Device Farm comes with a one-time free trial of 1000 device minutes. After that you are charged $0.17 per device minute. For more information about pricing, see [AWS Device Farm Pricing](https://aws.amazon.com/device-farm/pricing/). 
+{{% /notice %}}
+
+## Start Device Farm
+1. Go to [AWS Device Farm](https://us-west-2.console.aws.amazon.com/devicefarm/home?region=us-east-1#/projects).
```
Comment (JoeShi) [inline content/getting-started/device-farm-tutorial.md]: the entire workshop use us-east-1 region.

File: content/getting-started/device-farm-tutorial.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+1. Enter project name `alexaworkshop`, select **Create project**.
+
+1. After the creation you will access to the project's console, on the **Remote access** tab, select **Create project**.
+    ![deviceFarmConsole](/images/getting-started/device-farm-console.png)
+
+1. Under **Choose a device**, you can choose any Android device whose status is **AVAILABLE**, here we choose `Google Pixel 2`. After the creation of session, you will see the Device requested dialog box, then you can interact with this device.
+
+    {{% notice info %}}
+The duration of a remote access session is 150 minutes, after that the session will be closed and can not be used again.
+If you should close the browser or browser tab without stopping the session or if the connection between the browser and the internet is lost, the session remains active for five minutes. After that, Device Farm ends the session.
```
Comment (JoeShi) [inline content/getting-started/device-farm-tutorial.md]: If you should? what is the meaning?

File: content/getting-started/device-farm-tutorial.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+---
+title: "Use Alexa App on Device Farm (optional)"
+date: 2019-09-26T13:29:22+08:00
+draft: false
+chapter: false
```
Comment (JoeShi) [inline content/getting-started/device-farm-tutorial.md]: Have you tried to run this locally? need `weight` parameter I think.

File: content/getting-started/device-farm-tutorial.md (hunk 1 above)
Comment (fanyizhe) [reply content/getting-started/device-farm-tutorial.md]: Device farm is global and not associated with a region.

File: content/getting-started/device-farm-tutorial.md (hunk 2 above)
Comment (fanyizhe) [reply content/getting-started/device-farm-tutorial.md]: I will change it.

File: content/getting-started/device-farm-tutorial.md (hunk 3 above)
Comment (fanyizhe) [reply content/getting-started/device-farm-tutorial.md]: Yes, and I think `weight` is not necessary.

Comment (JoeShi) [reply content/getting-started/device-farm-tutorial.md]: If you do not include `weight` parameter, then the left side navigation bar will rank the page according to the alphabet.. Which is not what we wanted. Put this article under [BIND ECHO DEVICE (OPTIONAL)]

Comment (fanyizhe) [reply content/getting-started/device-farm-tutorial.md]: Done.

**Pull request merged**